test/lib/RuleManager.test.ts

```diff
@@ -175,7 +187,7 @@ describe('RuleManager', () => {
 				process.chdir(path.join('test', 'code-fixtures', 'projects'));
 			});
 			after(() => {
-				process.chdir("../..");
+				process.chdir("../../..");
```

Comment:
This might be the same issue that Josh ran into last week with the junit output.

Reply:
It is.